pkg/commands/auth/actions/login.go

```diff
@@ -91,29 +98,23 @@ func Login() *cobra.Command {
 			}
 
 			eg, ctx := errgroup.WithContext(cmd.Context())
+			if personalToken == "" {
+				loginViaBrowser(eg, ready, cmdFlags, ctx)
```

Comment:
this function dont return any error which it should and you not catching the error and returning it